_functions_onlyoffice.sh

```diff
@@ -75,6 +75,15 @@ do_start_onlyoffice() {
     return
   fi
 
+  # add onlyoffice security paramaters to exo server conf
+  if [ -z "${DEPLOYMENT_ONLYOFFICE_SECRET}" ]
+  then
+  echo_error "Missing DEPLOYMENT_ONLYOFFICE_SECRET configured environment variable. OnlyOffice may not be deployed"
+    exit 1
+  fi 
+  
+  export ${DEPLOYMENT_OPTS}="${DEPLOYMENT_OPTS} -Donlyoffice.documentserver.accessOnly=false -Donlyoffice.documentserver.secret=${DEPLOYMENT_ONLYOFFICE_SECRET}"
```

Comment:
It's not a correct syntax

```suggestion
  export DEPLOYMENT_OPTS="${DEPLOYMENT_OPTS} -Donlyoffice.documentserver.accessOnly=false -Donlyoffice.documentserver.secret=${DEPLOYMENT_ONLYOFFICE_SECRET}"
```